### ISSUE TITLE
Proxy by country?

### ISSUE THREAD

**Issue by danipolo** [all-bear/nodejs.proxy-rotator#6]
Hi,

Is possible to filter the proxies by country?

**Comment by all-bear** [comment]
@danipolo Hi, for now now, but I think it can be implemented. 
https://www.npmjs.com/package/proxy-lists is used to get proxy list and as I can see from docs it supports  proxies by country

```
var ProxyLists = require('proxy-lists');
 
var options = {
    countries: ['us', 'ca']
};
 
// `gettingProxies` is an event emitter object.
var gettingProxies = ProxyLists.getProxies(options);
```

so we can use this option inside of https://github.com/all-bear/nodejs.proxy-rotator/blob/master/lib/proxy-provider.js#L171

you can create pull request for this, or if you will have some problems with this, please ping me then I'll do this during few days

**Comment by strarsis** [comment]
@all-bear: Yes, being able to configure the proxy would be great, especially the countries option.
Any plans for adding this feature?
